config.json

```diff
@@ -78,6 +78,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "Coverage": {
```

Comment:
Is this needed at all? I don't see you using it at all.

Reply:
I'm using it to turn on coverage from build.cmd

Reply:
Oh yeah missed that parameter since you where just calling dotnet msbuild directly to do the coverage report.